doc/md/versioned-migrations.mdx

```diff
@@ -152,6 +152,12 @@ import (
 
 func main() {
 	ctx := context.Background()
+	
+	// Check if the migration directory exists and create it otherwise
+	if _, err := os.Stat(migrationDirPath); os.IsNotExist(err) {
+		os.Mkdir(migrationDirPath, 0755)
+	}
```

Comment:
`os.Mkdir` requires another error handling. Better to tell the user to create it manually in the first step of the tutorial rather than having it in the script.

Reply:
changed